### PR TITLE
fix(es/transforms/module): transform esmodule import to sequence expression

### LINKED ISSUE #1556: Transform esmodule without break `this`
**Describe the bug**
As we have discussed in #1524, when swc transform esmodule to `commonjs|amd|umd` it will generate code like
```js
const a = require('./a')
a.foo()
```
However, that will make `this` in `foo` points to module `a` instead of `globalThis`. Although few people still use commjs target today and even fewer people use `this` to refer global, this still should be fixed. swc should instead generate code like
```js
const a = require('./a')
(0, a).foo()
```
**Version**
latest

**Additional context**

### PR DESCRIPTION
**Pull request by Austaras**
Fixes #1556

### CODE REVIEW

File: ecmascript/transforms/react/src/refresh/mod.rs [hunk 1]
```diff
@@ -790,15 +794,15 @@ impl<C: Comments> Fold for Refresh<C> {
             return module_items;
         }
 
-        module_items.visit_with(&Invalid { span: DUMMY_SP } as _, self);
+        self.visit_module_items(&module_items, &Invalid { span: DUMMY_SP } as _);
```
Comment (kdy1) [inline ecmascript/transforms/react/src/refresh/mod.rs:797]: Why did you change this? Seems identical to me.

**Review by kdy1 — COMMENTED** [review]
Mostly LGTM.
I found one tiny problem, though.

File: ecmascript/transforms/react/src/refresh/mod.rs (hunk 1 above)
Comment (Austaras) [reply ecmascript/transforms/react/src/refresh/mod.rs:797]: To save one import. I believe these two are identical.

Comment (kdy1) [reply ecmascript/transforms/react/src/refresh/mod.rs:797]: I see. Thank you!

**Pull request merged**